main/rsyslog/APKBUILD

```diff
@@ -104,6 +104,15 @@ snmp() {
 		"$subpkgdir"/usr/lib/rsyslog/
 }
 
+elasticsearch() {
+	pkgdesc="rsyslog elasticsearch support"
+	makedepends="curl"
```

Comment:
`makedepends` must not be declared inside a split function.

Reply:
thx -- I thought I had read this would be the correct way to do it. Now I know better :-) Will correct soon, I have seen one other thing also glitched through.